tracer/src/Datadog.Trace/LibDatadog/Error.cs

```diff
@@ -0,0 +1,32 @@
+// <copyright file="Error.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace Datadog.Trace.LibDatadog;
+
+[StructLayout(LayoutKind.Sequential)]
+internal struct Error
```

Comment:
If this matches a struct (or equivalent) in `libdatadog`, can we add a comment with a link to that struct?

Reply:
not asking for this specific struct, but in general. is this something we should be doing for structs and  method calls (p/invokes)?